File: src/main/java/com/hedvig/backoffice/services/members/dto/PersonDTO.java
package com.hedvig.backoffice.services.members.dto;

import com.hedvig.backoffice.graphql.types.Whitelisted;
import lombok.Value;

import java.util.List;
import java.util.Optional;

@Value
public class PersonDTO {
  List<Flag> personFlags;
  DebtDTO debt;
  Optional<Whitelisted> whitelisted;
  PersonStatusDTO status;
}

File: src/main/java/com/hedvig/backoffice/services/members/dto/PersonDTO.kt
package com.hedvig.backoffice.services.members.dto

import com.hedvig.backoffice.graphql.types.Whitelisted

data class PersonDTO(
    val flags: PersonFlags,
    val debt: DebtDTO,
    val whitelisted: Whitelisted?,
    val status: PersonStatusDTO
)
